gwsumm/data/utils.py

```diff
@@ -29,16 +29,12 @@
 from ligo.segments import segmentlist as LigoSegmentList
 
 from gwpy.segments import (DataQualityFlag, SegmentList, Segment)
-from gwpy.signal.fft import (get_default_fft_api, lal as fft_lal)
-from gwpy.signal.fft.registry import get_method as get_fft_method
 
 from ..channels import get_channel
 from ..config import GWSummConfigParser
 
 __author__ = 'Duncan Macleod <duncan.macleod@ligo.org>'
 
-FFT_API = get_default_fft_api()
-
 # get FFT scheme
 FFT_SCHEME = None
 if FFT_API.startswith('pycbc'):
```

Comment:
I think you need to remove this whole block, because `FFT_API` has been removed.